odmltools/importers/import_datacite.py

```diff
@@ -0,0 +1,571 @@
+"""odmlImportDatacite
+
+Convenience script to parse a datacite XML file or whole directories
+containing datacite XML files and create odML files with the parsed
+information.
+
+Usage: odmlimportdatacite [-f FORMAT] [-o OUT] [-r] [-p] INPUT
+
+Arguments:
+    INPUT    Path and filename of the datacite XML file to be parsed.
+             If used with the [-r] flag, INPUT should be a directory;
+             all datacite XML files within this directory and any sub
+             directories will be parsed to odML.
+
+Options:
+    -f FORMAT   odML output file format. Available formats are
+                  'XML', 'JSON', 'YAML', 'RDF'. Default format is 'XML'.
+    -o OUT      Output directory. Must exist if specified.
+                  If not specified, output files will be written to the
+                  current directory.
+    -r          [Optional] Walk recursively through a repository.
+                  and convert all datacite files found.
+    -p          [Optional] Print the parsed document tree(s) to the command line.
+                   Default is False.
+    -h --help   Show this screen.
+    --version   Show version number.
+"""
+
+import os
+import pathlib
+import re
+import sys
+
+from datetime import date
+from xml.parsers.expat import errors as exp_err
+from xml.parsers.expat import ExpatError
+
+import xmltodict
+
+from docopt import docopt
+from odml import Document, Section, Property
+from odml.fileio import save as save_odml
+from odml.dtypes import DType
+from odml.tools.parser_utils import SUPPORTED_PARSERS
+
+
+VERSION = "0.1.0"
```

Comment:
Should this be removed so that there's only a global package version (1.0.0 from the info.json)?